### PR TITLE
Doxygen Documentation: stdbool.h #23

### PR DESCRIPTION
**Pull request by seekaddo**
Doxygen documentation for the functions in stdbool.h. I have already tested it.

### CODE REVIEW

**Comment by seekaddo** [comment]
Hmmm same old prblem. Is there a way to make more PR's without this conflicting issues.

**Comment by phillipjohnston** [comment]
@seekaddo how are you making your branches? My usual workflow is follows two patterns:

1. start every new standalone branch from `master` instead of my previous branch (`git checkout  master; git checkout -b new_branch`)
2. Make many changes on a single branch. Once I'm ready to create PRs, I cleanup all my commits and merge them into standalone changes. Then I create new branches on master and use `git cherry-pick` to select the commit(s) I want on that particular branch

**Comment by phillipjohnston** [comment]
To easily undo this, you can either delete commit `7921e5d59162b55b35932b42ac9476ba1f5dcef8` during an interactive rebase.

**Comment by phillipjohnston** [comment]
Also, a tip: you can auto-close issues using commits and pull requests if you reference the number, such as "Closes #23" or "Fixes #23".

File: include/stdbool.h [hunk 1]
```diff
@@ -3,12 +3,17 @@
 
 #ifndef __cplusplus
 
-/**
-* Only define these for C, since C++ has its own bool support
-*/
+/*
+ * Only define these for C, since C++ has its own bool support
```
Comment (phillipjohnston) [inline include/stdbool.h]: Slight modification here: I would correct this to "Only defined for C, since C++ has its own bool support".

Comment (seekaddo) [reply include/stdbool.h]: Ok thanks. I will make those changes.

**Comment by seekaddo** [comment]
@phillipjohnston made and problem fixed. Thanks for your the tips. I will follow it to handle multiple PR's in the future. Merry Christmas and enjoy the holidays and happy weekend.

**Comment by phillipjohnston** [comment]
Thanks @seekaddo! I appreciate your help on this project.

I hope you have a great weekend and Christmas as well.  🎄 🍻

**Pull request merged**